src/homework_oop_v2/Truck.java

```diff
@@ -0,0 +1,20 @@
+package homework_oop_v2;
+
+public class Truck extends Transport implements Serviceable {
+
+    public Truck(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+    }
+
+    public void service() {
+        if (this != null) {
+            System.out.println("Обслуживаем " + this.getModelName());
```

Comment:
```suggestion
            System.out.println("Обслуживаем " + getModelName());
```
this можно не писать